xbmc/GUIInfoManager.cpp

```diff
@@ -6850,13 +6850,20 @@ const infomap container_str[]  = {{ "property",         CONTAINER_PROPERTY },
 ///     <p><hr>
 ///     @skinning_v19 **[New Infolabel]** \link ListItem_AlbumStatus `ListItem.AlbumStatus`\endlink
 ///   }
-///   \table_row3{   <b>`ListItem.HdrType`</b>,
-///                  \anchor ListItem_HdrType
-///                  _string_,
-///     @return String containing the name of the detected HDR type or empty if not HDR. See \ref StreamHdrType for the list of possible values.
-///     <p><hr>
-///     @skinning_v20 **[New Infolabel]** \link ListItem_HdrType `ListItem.HdrType`\endlink
-///   }
+///   \table_row3{   <b>`ListItem.HdrType`</b>,
+///                  \anchor ListItem_HdrType
+///                  _string_,
+///     @return String containing the name of the detected HDR type or empty if not HDR. See \ref StreamHdrType for the list of possible values.
+///     <p><hr>
+///     @skinning_v20 **[New Infolabel]** \link ListItem_HdrType `ListItem.HdrType`\endlink
+///   }
+///   \table_row3{   <b>`ListItem.SongVideoURL`</b>,
+///                  \anchor ListItem_SongVideoURL
+///                  _string_,
+///     @return youtube plugin link to a youtube video of a song
```

Comment:
Maybe better to use video URL and avoid mentioning YouTube, the add-on is responsible for setting the plugin://YouTube URL isn't it?

Reply:
Yeah, the scraper formats up the entire URL when it fetches the link(s).  So the listitem will contain something like `plugin://plugin.video.youtube/?action=play_video&videoid=someYTlink`.  I would have though in that case that the documentation is correct, but I can change it if you'd prefer ?

Reply:
I think it would be better changed. `SongVideoURL` info doesn't need to be necessarily a youtube plugin link just something Kodi can perceive as a video representative of the song - it just happens that youtube is the norm. But could simply be some other video url or a simple http link

Reply:
The artist scraper specifically fetches yt links because that's the only data theaudiodb is holding and it's the only online db that Kodi uses that actually holds any info like that. I understand why you might want it to be more generic as in just a url to somewhere but it can only ever be a youtube link as that is the only url data to fetch.

Reply:
I understand but media providers and addons (scrapers) in this case is not the same as the Kodi core. Kodi can store any string there (or any video to be more precise) its up to the addon to provide a youtube link. You could probably add (e.g. youtube) to the documentation but explicitly state it has to be a youtube link is somehow wrong